lnwallet/channel.go

```diff
@@ -5552,7 +5355,6 @@ func (lc *LightningChannel) ReceiveRevocation(revMsg *lnwire.RevokeAndAck) (
 			addIndex++
 
 			pd.isForwarded = true
-			addsToForward = append(addsToForward, pd)
```

Comment:
this should be attached to `fwdPkg.Add` instead

Reply:
Going back through this, the code isn't clear, but what you're asking for is already being done. I'll update to make it more explicit